dotnet/samples/GettingStartedWithProcesses/Step00/Step00_Processes.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.SemanticKernel;
+using Step00.Steps;
+
+namespace Step00;
```

Comment:
I'll defer to @alliscode to suggest ordering for this sample.  I very much appreciate including the sample with the code-change and incorporating feedback on the existing "default" events.